.gitignore

```diff
@@ -27,3 +27,5 @@ Studies/Visual
 /filesEntries.txt
 /filesEntries_SingleMuon2018A.txt
 /cmssw_report.xml
+Corrections/data/*/*/*.root
+*.txt
```

Comment:
there are too many entries in `.gitignore` keep only 
```
/.law
/data*
__pycache__
/output
/soft
/crab
/.crabOverseer*
```

Reply:
ok